docs/source/entities.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: Entities in Apollo Federation
+sidebar_title: Entities (basics)
+description: Resolve types across multiple subgraphs
+---
+
+import { CodeColumns, ExpansionPanel } from 'gatsby-theme-apollo-docs';
+
+> ⚠️ Some details of entity behavior have changed in Federation 2. For a summary of these changes, [see what's new](/federation-2/new-in-federation-2/).
+
+In a federated graph, an **entity** is an object type that defines its fields across multiple subgraphs. Each subgraph is responsible for resolving whichever fields it defines for an entity.
+
+Here, the `Product` entity's fields are defined across two subgraphs:
+
+<CodeColumns>
+
+```graphql
+# Products subgraph
+type Product @key(fields: "id") {
+  id: ID!
+  name: String!
+  price: Int
+}
+```
+
+```graphql
+# Inventory subgraph
+type Product @key(fields: "id") {
+  id: ID!
+  inStock: Boolean!
+}
+```
+
+</CodeColumns>
+
+Types _besides_ object types (unions, interfaces, etc.) _cannot_ be entities.
+
+## Defining an entity
+
+In a subgraph schema, you can designate any existing object type as an entity by adding the `@key` directive to its definition, like so:
```

Comment:
I worry a bit about that kind of sentence because that notion of a type being designated as an entity does not really exists in the code, and there is use cases (see this [this discussion](https://apollographql.quip.com/bQJMAqFttef4#cETADAQUhtS) for instance), for an entity type to have a `@key` in some subgraphs but not others.

Tbc, I don't have a super simple fix to offer. Currently, "entities" is more of a usage pattern than a real internal concept, and that makes it hard to have a simple characterisation of entities that is both simple, precise and always correct. This is why I initially advocate for adding an `@entity` annotation to "force" a clear definition in the model, but we went another way (and to be fair, that proposal was not perfect either).

Anyway, I just though of something that could hopefully clarify our story here but I need to think this through a bit more.  So this comment is probably not very actionable right now, but I still wanted to flag that there is a bit of vagueness in our concepts right now, and that's certainly unfortunate.